packages/nocms/src/lib/plugins.js

```diff
@@ -3,15 +3,15 @@ import path from 'path';
 
 export async function loadPlugins(pluginActivationContext) {
 	return Promise.all([
-		findFilesAsync('node_modules/nocms-plugin-*/', {})
+		findFilesAsync(`${process.cwd()}/node_modules/nocms-plugin-*/`, {})
```

Comment:
This is only true for development, right?

Reply:
No, this should be true for all runs, since the `process.cwd()` is where the `nocms` command is run, for instance standalone on the command line or initiated through the npm scripts section

Reply:
Please use `path.join` :)

Reply:
> Please use `path.join` :)

we're using `path` later, we're just preparing the pattern here

Reply:
Correction: We can totally use `path.join`, no need to resolve the entire path, PR will be updated soon